cgotorch/cgotorch.h

```diff
@@ -60,6 +60,7 @@ void FreeString(const char *s);
 void Tensor_Backward(Tensor a);
 Tensor Tensor_Grad(Tensor a);
 
+void ManualSeed(int seed);
```

Comment:
we can merge #115 and then rebase this PR.

Reply:
OK. I merged #115. However, there comes some conflicts.